atlasdb-impl-shared/src/test/java/com/palantir/atlasdb/transaction/impl/expectations/TrackingKeyValueServiceReadInfoTest.java

```diff
@@ -375,14 +265,4 @@ private void validateReadInfoForLazyRead(int size) {
         assertThat(trackingKvs.getOverallReadInfo()).isEqualTo(readInfo);
         assertThat(trackingKvs.getReadInfoByTable()).containsOnlyKeys(ImmutableList.of(tableReference));
     }
-
-    private void validateReadInfoForTableAgnosticRead(String methodName, int size) {
-        TransactionReadInfo readInfo = ImmutableTransactionReadInfo.builder()
-                .bytesRead(size)
-                .kvsCalls(1)
-                .maximumBytesKvsCallInfo(ImmutableKvsCallReadInfo.of(methodName, size))
-                .build();
-        assertThat(trackingKvs.getOverallReadInfo()).isEqualTo(readInfo);
-        assertThat(trackingKvs.getReadInfoByTable()).isEmpty();
-    }
```

Comment:
Alright I'll revert

Reply:
Discussed: these tests have been removed because the underlying interface that the class implements changed, and the new interface does not contain these. So these are being removed correctly

Reply:
yep! We still expose those at the concrete impl, but not here - fair enough